app/src/main/java/org/fossasia/susi/ai/helper/Constant.java

```diff
@@ -37,7 +37,7 @@
     public static final String SELECT_SERVER = "Server_Select";
     public static final String LANG_SELECT = "Lang_Select";
     public static final String LANGUAGE = "prefLanguage";
-    public static final String DEFAULT = "default";
+    public static final String DEFAULT = "en";
```

Comment:
@codedsun this is the reason for the crash on opening settings

Reply:
@batbrain7 @codedsun Thanks for pointing out this issue. I see this wasn't reviewed well. I request you to please review the PRs properly before they are merged. Also, there wasn't any crash while this was fixed. In fact, I received a crash when I kept default rather than en. I was asked to change the constant rather than making changes everywhere. And, after doing that too there was no crash. So, it would be great if you review the code well and understand that without proper testing, there are always higher chances of getting bugs and crashes, and please don't approve any such PR where you have a doubt.